apps/snapodds/src/index.html

```diff
@@ -58,17 +58,59 @@
         );
       }
 
-      SnapoddsSdk.sportsMediaBuilder()
+      const result = {
```

Comment:
These lines look temporal to me, required only for this feature development. If so, I would prefer them to be deleted.

Reply:
Ok